desktop-app/src/renderer/components/Previewer/Device/index.tsx

```diff
@@ -531,7 +535,10 @@ const Device = ({ isPrimary, device, setIndividualDevice }: Props) => {
   const scaledWidth = width * zoomfactor;
 
   const isRestrictedMinimumDeviceSize =
-    device.width < 400 && zoomfactor < 0.6 && !rotateDevices && !singleRotated;
+    device.width < 400 &&
+    zoomfactor < 0.6 &&
+    !(rotateDevices && isDeviceRotatable) &&
```

Comment:
For readability sake, we can have one variable called `isDeviceRotationEnabled` and use that in both these conditionals.